src/test/deploy/functions/prompts.spec.ts

```diff
@@ -87,11 +98,12 @@ describe("promptForFailurePolicies", () => {
   });
 
   it("should propmt if an existing function adds a failure policy", async () => {
-    const func: CloudFunctionTrigger = {
+    const func = {
       name: "projects/a/locations/b/functions/c",
       entryPoint: "",
       labels: {},
       environmentVariables: {},
+      runtime: "nodejs14" as gcf.Runtime,
```

Comment:
nit: Is this cast required? If so, I think I might prefer a plain string union type to a separately declared type. Having to cast string literals is kinda annoying.

Reply:
This is a case of type forgetting. If we called this a gcf.CloudFunction then the literal string would compile completely OK, but we're currently using one literal to be both an `Omit<gcf.CloudFunction, gcf.OutputOnlyFields>` and as a `CloudFunctionTrigger` and the literal type just rembers "nodejs14" a string.

The final version of this code uses a `backend.FunctionSpec` for both input and output functions and the literal "nodejs14" is validated and remembered to be a `backend.Runtime` type.